Changes

```diff
@@ -1,5 +1,70 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v8.2.0 2021-12-20 (public fix release)
+
+ [Deprecation]
+ - The use of language tags that includes country code is
+   deprecated. See "docs/API.md#language-tag". Such
+   language codes will be made illegal in the v2022.2 release.
+   Use language codes without country code instead.
+ - The use of an empty string in the "LANGUAGE.locale"
+   setting is deprecated and will be made illegal in the
+   v2022.2 release. See "docs/Configuration.md#language-section".
+ - The API methods "get_test_results" and "get_test_history"
+   returns two keys with the same timestamp in different
+   formats. The "creation_time" key is deprecated and will
+   be removed with release v2023.1. Use the "created_at"
+   key instead. See "docs/API.md#api-method-get_test_results"
+   and "docs/API.md#api-method-get_test_history".
+ - API method "get_test_results" returns key "id", which is now
+   deprecated and will be removed in v2022.2 release. See
+   "docs/API.md#api-method-get_test_results". (#949)
+ - Removes deprecated configuration options (#954)
+
+ [Breaking changes]
+ - Domain names cannot be entered with consecutive tailing
+   dots (#983)
+
+ [Features]
+ - Adds support for CentOS Linux 7 again - to be removed again
+   with release v2023.1 (#1027)
+ - Updates zmb for "start_domain_test" and "add_batch_job" to include
+   support for "queue"(#1006, #1002)
```

Comment:
Missing space between `"queue"` and following parenthesis

Reply:
Fixed.